one-pager-maker/src/entity/utils.ts

```diff
@@ -1,4 +1,12 @@
-import {Timestamp, FieldValue} from "firebase/firestore";
+import {FieldValue, Timestamp} from "firebase/firestore";
+
+export type ForCreate<T> = Omit<T, "id" | "created_at" | "updated_at">;
+export type ForCreateWithId<T> = Omit<T, "created_at" | "updated_at">;
```

Comment:
Is `ForCreateWithId`  unused type?  
Or do you plan to use it for the next implementation?

Reply:
Maybe I will use it when I add UserRepository.#create() because we will explicitly 
specify the id of user data.